next/components/deck-card-table-row.js

```diff
@@ -46,25 +58,60 @@ export default function DeckCardsTableRow({ card, deleteCard, quantity }) {
           visibility: hidden;
           opacity: 0;
         }
+        .deck-card-quantity {
+          width: 48px;
```

Comment:
Can we make this 30px? It doesn't interfere with the deck page and it makes the deck builder more compact (we're gonna need that extra space for later stories)

```suggestion
          width: 30px;
```